src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs

```diff
@@ -92,14 +97,32 @@ public partial class MediaManager : IDisposable
 		{
 			Player = Player
 		};
-
 		// Pre-initialize Volume and Muted properties to the player object
 		Player.Muted = MediaElement.ShouldMute;
 		var volumeDiff = Math.Abs(Player.Volume - MediaElement.Volume);
 		if (volumeDiff > 0.01)
 		{
 			Player.Volume = (float)MediaElement.Volume;
 		}
+		if (Dispatcher.IsDispatchRequired)
+		{
+			Dispatcher.Dispatch(() =>
+			{
+				UIApplication.SharedApplication.BeginReceivingRemoteControlEvents();
+			});
+		}
+		else
+		{
+			UIApplication.SharedApplication.BeginReceivingRemoteControlEvents();
+		}
+#if IOS
+		PlayerViewController.UpdatesNowPlayingInfoCenter = false;
+#else
+		PlayerViewController.UpdatesNowPlayingInfoCenter = true;
+#endif
+		var avSession = AVAudioSession.SharedInstance();
```

Comment:
In the documentation we have on line 183 and 184 a reference to code I have implemented that is required for notifications and lock screen content. Do we want to revise or remove that code as it would end up duplicating what I have already done? It could cause issues. There is also nothing in documentation about cleaning up after you don't need it anymore either? It needs to be released and there is not mention of that at all. Releasing it is as shown below is required for proper cleanup.
```
var audioSession = AVAudioSession.SharedInstance();
				audioSession.SetActive(false);
```

Reply:
If the code you mentioned is already on the code and it is not required anymore. Then I recommend to remove it from the documentation, since it is not a requirement.

Reply:
I think we should opt to giving the developer more control over it as per https://github.com/CommunityToolkit/Maui/discussions/1846

Reply:
If that code is removed support for metadata and lockscreen will be disabled on iOS and Mac. I can safely remove it and allow developers to add it themselves but it needs to be added during startup. Or I can add another bindable property that can be used to toggle it off and on. What do you suggest? I will have to experiment and see exactly how we can implement that sort of change.

Reply:
Let's leave it out of this PR. I'm happy to provide a PR/proposal that shows my thinking.

Reply:
What specifically do you want removed? Is this what you want removed or a subset of this? 
```csharp
if (Dispatcher.IsDispatchRequired)
		{
			Dispatcher.Dispatch(() =>
			{
				UIApplication.SharedApplication.BeginReceivingRemoteControlEvents();
			});
		}
		else
		{
			UIApplication.SharedApplication.BeginReceivingRemoteControlEvents();
		}
#if IOS
		PlayerViewController.UpdatesNowPlayingInfoCenter = false;
#else
		PlayerViewController.UpdatesNowPlayingInfoCenter = true;
#endif
		var avSession = AVAudioSession.SharedInstance();
		avSession.SetCategory(AVAudioSessionCategory.Playback);
		avSession.SetActive(true);
```